skimage/_shared/tests/test_utils.py

```diff
@@ -1,7 +1,22 @@
-from skimage._shared.utils import copy_func
+from skimage._shared.utils import (copy_func, 
+                                    assert_nD)
 import numpy.testing as npt
+import unittest
 
 
+
+def test_assert_nD():
+    try:
+        z = np.random.random(200**2).reshape((200, 200))
+        x = z[10:30, 30:10]
+        assert_nD(x, 2)
```

Comment:
There is no need to use `try/except` here. Just use `numpy.testing.assert_raises`, i.e.
```
from numpy.testing import assert_raises
...
assert_raises(ValueError, assert_nD, x, 2)
```